doc/tutorials/cmerge.md

```diff
@@ -209,6 +209,12 @@ kdb get user:/tests/arrays/result/#0
 #> previous
 ```
 
+## Cleanup
+
+```sh
+kdb rm -rf user:/tests
```

Comment:
Why -f if there is a namespace specified?

Reply:
That was a mistake, I removed it.